EDI/UserInterface/01a_Exports.md

```diff
@@ -0,0 +1,340 @@
+
+- [CANCEL]    
+   Click this button to finish the wizard without saving it.    
+   The *Select source* wizard window is closed.
+
+- [CONTINUE]   
+   Click this button to go one step forward.    
+   The *Map attributes* wizard window is displayed (see below). 
+
+### Step 2: Map attributes
+
+*DataHub Exporter > Exports > Select export definition > Tab Data sources > Click an exporter definition*
+
+*DataHub Exporter > Exports > Select export definition > Tab Data sources > Click the Add button > Click the CONTINUE button*
+
+![Map attributes](../../Assets/Screenshots/EDI/UserInterface/MapAttributes.png "[Map attributes)]")
+
+This step is displayed when you assign a data source. It is also displayed when you select a data source for doing the mapping. For detailed information on the mapping functions, see [Mapping](../../DataHub/UserInterface/02d_ETL.md#mapping) in the *DataHub* documentation.
+
+>[Info] Note that the wizard does not provide the whole functional scope that the *DataHub* mapping provides. The wizard contains only those functions that are relevant to export message mappings.
+
+- [CANCEL]    
+   Click this button to finish the wizard without saving it.    
+   The wizard window is closed.
+
+- [CONTINUE]   
+   Click this button to go one step forward.    
+   The *Optional: Filter items* wizard step is displayed (see below). 
+ 
+- [< BACK]   
+  Click this button to return to the previous step.
+
+### Step 3: Optional: Filter items
+
+*DataHub Exporter > Exports > Select export definition > Tab Data sources > Click an exporter definition > Click the CONTINUE button to step 3*
+
+*DataHub Exporter > Exports > Select export definition > Tab Data sources > Click the CONTINUE button to step 3*
+
+![Optional: Filter items](../../Assets/Screenshots/EDI/UserInterface/OptionalFilterItems.png "[Optional: Filter items]")
+
+This step is currently not in use. In general, it is only relevant for mass exports.
+
+- [CANCEL]    
+   Click this button to finish the wizard without saving it.   
+   The wizard window is closed.
+
+- [CONTINUE]   
+   Click this button to go one step forward.    
+   The *Select additional data* wizard step is displayed (see below). 
+ 
+- [< BACK]   
+  Click this button to return to the previous step.
+
+
+### Step 4: Select additional data
+
+*DataHub Exporter > Exports > Select export definition > Tab Data sources > Click an exporter definition > Click the CONTINUE button to step 4*
+
+*DataHub Exporter > Exports > Select export definition > Tab Data sources > Click the CONTINUE button to step 4*
+
+![Select additional data](../../Assets/Screenshots/EDI/UserInterface/SelectAdditionalData.png "[Select additional data]")
+
+Use this step to enter additional data such as the limit of line items to be output.
+
+- *Limit*    
+   When you currently specify the header definition settings: Enter the maximum number of line items that you want to output per message.
```

Comment:
Vorschlag: Enter the maximum number of line items that you want to output per message. This value is only relevant if you want to specify the header definition settings.

Reply:
Ich würde immer die wentliche Information an den Anfang stellen und nicht ans Ende.